src/core/trulens/core/utils/imports.py

```diff
@@ -151,73 +135,18 @@ def get_package_version(name: str) -> Optional[version.Version]:
     pip install '{req}'
     ```
 
-If you are running trulens_eval in a notebook, you may need to restart the
+If you are running trulens in a notebook, you may need to restart the
 kernel after resolving the conflict. If your distribution is in a bad place
-beyond this package, you may need to reinstall trulens_eval so that all of the
+beyond this package, you may need to reinstall trulens so that all of the
 dependencies get installed and hopefully corrected:
 
     ```bash
-    pip uninstall -y trulens_eval
-    pip install trulens_eval
+    pip uninstall -y trulens
+    pip install trulens
     ```
 """
 
```

Comment:
removed check_imports from `__init__` as most hard dependencies should be resolved at install-time so this no longer needed.